test/games/planets_zebeth/exporter/test_planets_zebeth_game_exporter.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import annotations
+
+import multiprocessing
+from concurrent.futures import Future
+from contextlib import contextmanager
+from typing import TYPE_CHECKING
+from unittest.mock import ANY, MagicMock
+
+import planets_yapr.wrapper
+import pytest
+
+from randovania.games.planets_zebeth.exporter.game_exporter import (
+    PlanetsZebethGameExporter,
+    PlanetsZebethGameExportParams,
+    _run_patcher,
+)
+from randovania.patching.patchers.exceptions import UnableToExportError
+
+if TYPE_CHECKING:
+    from collections.abc import Callable
+    from multiprocessing.connection import Connection
+    from pathlib import Path
+
+
+def test_export_game_raises_without_dotnet(mocker):
+    mocker.patch("subprocess.run", side_effect=FileNotFoundError)
+
+    exporter = PlanetsZebethGameExporter()
+
+    with pytest.raises(UnableToExportError):
+        exporter._do_export_game(
+            MagicMock(),
+            PlanetsZebethGameExportParams(spoiler_output=None, input_path=None, output_path=None),
+            MagicMock(),
+        )
+
+
+def test_export_game_raises_with_wrong_dotnet_exit_code(mocker):
+    dotnet_process = mocker.patch("subprocess.run")
+    dotnet_process.returncode = 1
+
+    exporter = PlanetsZebethGameExporter()
+
+    with pytest.raises(UnableToExportError):
+        exporter._do_export_game(
+            MagicMock(),
+            PlanetsZebethGameExportParams(spoiler_output=None, input_path=None, output_path=None),
+            MagicMock(),
+        )
+
+
+@pytest.mark.parametrize("patch_data_name", ["starter_preset", "starter_preset_shuffle_keys"])
+def test_do_export_game(test_files_dir, mocker, patch_data_name: str, tmp_path):
```

Comment:
I'm not quite understanding what this test is trying to do. From what it looks like to me is that its actually trying to call the patcher?

Reply:
don't I need to test calling the patcher?

Reply:
Okay nevermind. 
I reread this now while not being tired, seems ok.